chia/data_layer/s3_plugin_service.py

```diff
@@ -14,7 +14,7 @@
 from typing import Any, Optional, overload
 from urllib.parse import urlparse
 
-import boto3 as boto3
+import boto3
```

Comment:
I presume we don't want it here, but note that this does trigger mypy to consider it as exported so it isn't entirely without 'purpose'.  Glad to see it removed here though.